entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -130,7 +114,7 @@ class EntityService(
         entity: Entity,
         propertyKey: String,
         propertyValues: Map<String, List<Any>>
-    ): Property {
+    ): String {
         val propertyValue = getPropertyValueFromMap(propertyValues, NGSILD_PROPERTY_VALUE)

```

Comment:
Not related to improvement but readibility, maybe we could move the building part of the Property/Relationship in a builder ?

Reply:
Or even at least in constructors or an Util class

Reply:
Also